.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+python:
+  # We don't actually use the Travis Python, but this keeps it organized.
+  - '3.7'
+install:
+  - sudo apt-get update
+  # We do this conditionally because it saves us some downloading if the
+  # version is the same.
+  wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
```

Comment:
This is just a simple parser; we shouldn't have to depend on conda (60M) for testing it.

Reply:
updated